Add cleaner spray to medical robots

Ninetailed opened the pull request:
It just seems weird that surgeon bot has no way of cleaning up their OR and the crisis bot is stuck with a filthy infirmary, while so many other modules get free space cleaner.

You're on your own for refilling the bottles, though. No one else gets free cleaner at recharge, and if janibot doesn't, why should medibots?

:cl: Ninetailed
rscadd: Surgeon, Crisis, and Emergency modules now have space cleaner
rscadd: Surgeon module now has sterilizine
tweak: Examining spray bottles now shows a message more like the ones beakers have.
/:cl:

Spookerton commented on code/modules/reagents/reagent_containers/spray.dm:
```diff
@@ -78,8 +78,13 @@
+	if(distance > 0)
+		return
+
+	if(reagents && reagents.reagent_list.len)
```
```suggestion
	if (reagents?.total_volume)
```

Ninetailed replied:
I always forget the conditional call operator exists. 🤦

Ninetailed committed:
Add cleaner spray to medical robots

It just seems weird that surgeon bot has no way of cleaning up their OR and
the crisis bot is stuck with a filthy infirmary, while so many other
modules get free space cleaner.

* Surgeon, Crisis, and Emergency modules now have space cleaner
* Surgeon module now has sterilizine
* Change spray bottles' examine message to be more consistent with beakers'
